kasa/discover.py

```diff
@@ -273,9 +274,27 @@ async def discover_single(
         """
         loop = asyncio.get_event_loop()
         event = asyncio.Event()
+
+        try:
+            ipaddress.ip_address(host)
+            ip = host
+        except ValueError:
+            try:
+                adrrinfo = await loop.getaddrinfo(
+                    host,
+                    0,
+                    type=socket.SocketKind.SOCK_DGRAM,
```

Comment:
```suggestion
                    type=socket.SOCK_DGRAM,
```
While the internal presentation is from `SocketKind` and `AddressFamily`, I think the preferred way is to use them like this?

Reply:
Done